.github/workflows/e2e-azure-management-and-workload-cluster.yaml

```diff
@@ -62,6 +62,7 @@ jobs:
 
       - name: Run Azure Management and Workload Cluster E2E Tests
         env:
+          DAILY_BUILD: ${{ github.event.client_payload.daily-build }}
```

Comment:
I don't believe it should... hahah! The daily build event is the only thing that populates that ENV value. If this doesn't work like I think it would, then we can always revert this.